pom.xml

```diff
@@ -71,16 +69,16 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
-            <dependency>
-                <groupId>io.gravitee.gateway.handlers</groupId>
-                <artifactId>gravitee-gateway-handlers-api</artifactId>
-                <version>${gravitee-gateway.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>io.gravitee.gateway</groupId>
-                <artifactId>gravitee-gateway-reactor</artifactId>
-                <version>${gravitee-gateway.version}</version>
-            </dependency>
+<!--            <dependency>-->
+<!--                <groupId>io.gravitee.gateway.handlers</groupId>-->
+<!--                <artifactId>gravitee-gateway-handlers-api</artifactId>-->
+<!--                <version>${gravitee-gateway.version}</version>-->
+<!--            </dependency>-->
+<!--            <dependency>-->
+<!--                <groupId>io.gravitee.gateway</groupId>-->
+<!--                <artifactId>gravitee-gateway-reactor</artifactId>-->
+<!--                <version>${gravitee-gateway.version}</version>-->
+<!--            </dependency>-->
```

Comment:
Forgot to delete this part ?

Reply:
Yep it was clean up.. forgot to remove. THanks